docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,872 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Prebuilt {anomaly-jobs}
+
+{anomaly-jobs-cap} contain the configuration information and metadata necessary 
+to perform an analytics task. {kib} can recognize certain types of data and 
+provide specialized wizards for that context. This page lists the available 
+prebuilt {anomaly-jobs} that you can use via {kib}. Every module item in the 
+list contains a name, a short description, information about 
+the {ml-docs}/ml-buckets.html#ml-buckets[`bucket_span`] and the 
+{ml-docs}/ml-functions.html[analysis function] as well as the influencers – the 
+entities that have contributed to, or are to blame for, the anomalies – of the 
+{anomaly-detect} job.
+
+
+[float]
+[[ootb-ml-jobs-apache]]
+=== Apache
+
+These {anomaly-job} wizards appear in {kib} if you use 
+{filebeat-ref}/index.html[{filebeat}] to ship access logs from your 
+https://httpd.apache.org/[Apache] HTTP servers to {es} and store it using fields 
+and datatypes from the Elastic Common Schema (ECS).
+
+low_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the event rate of HTTP requests. 
+* Detects unusually low counts of HTTP requests compared to the previous event 
+rate.
+
+This job can be created if web access log data exists matching the defined 
+filter.
+
+Bucket span: 15m.
+
+Function: `low_count`.
+
+
+source_ip_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high request rates in the HTTP access log 
+  compared to the previous rate.
+
+This job can be created if web access log data exists matching the defined 
+filter.
+
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_count`.
+
+
+source_ip_url_count_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high distinct count of URLs in the HTTP 
+access log.
+
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_distinct_count`.
+
+
+status_code_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the occurrences of HTTP response status codes (`partition_field_name` 
+  is `http.response.status_code`).
+* Detects unusual status code rates in the HTTP access log compared to previous 
+  rates.
+
+Influencers:
+
+* `http.response.status_code` 
+* `source.address`
+
+Bucket span: 15m.
+
+Function: `count`.
+
+
+visitor_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models visitor rates.
+* Detects unusual visitor rates in the HTTP access log ompared to previous 
+  rates.
+
+Bucket span: 15m.
+
+Function: `non_zero_count`.
+
+
```

Comment:
If we do start using these tagged regions from other books, however, we'll need to make sure our links work externally too.
```suggestion
end::apache-jobs[]
```